content/docs/tech-insights/track-react-version/index.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Tracking Dependency versions
+publishedDate: '2024-01-08T15:15:00.0Z'
+description: How to track and visualize the dependency versions of your software with Roadie Tech Insights
+---
+
+Tracking the dependency version is critical for ensuring your software is secure and compliant. It can be difficult to track if your software is using a supported version, and which are using an End of Life version.
```

Comment:
maybe
 ```suggestion
Tracking the dependency version is critical for ensuring your software is secure and compliant. It can be difficult to track which of your software is using a supported version, and which are using an End of Life version.
```
?